raddb/mods-available/imap

```diff
@@ -0,0 +1,159 @@
+		#  is not known ahead of time (using a URL from an external source), and/or the CA used
+		#  to sign the HTTPS server certificate is unknown.
+		#
+#		ca_path	             = "${certdir}"
+
+		#
+		#  certificate_file:: PEM formatted file containing the certificate we present to the HTTPS server
+		#
+		#  Specifies a certificate and any intermediary CAs we should send to the HTTPS server.
+		#
+		#  This file should usually contain the client certificate file first, then any
+		#  intermediary signing CAs, shallowest (direct signee of the certificate_file)
+		#  to deepest (signed directly by the root CA).
+		#
+#		certificate_file     = /path/to/radius.pem
+
+		#
+		#  private_key_file:: PEM formatted file containing the private key for the specified certificate_file
+		#
+		#  Must be specified if certificate_file is being used.
+		#
+#		private_key_file     = /path/to/radius.key
+
+		#
+		#  private_key_password:: Password used to decrypt the private key file.
+		#
+		#  Should only be specified in the private_key_file is encrypted.
+		#
+#		private_key_password = "supersecret"
+
+		#
+		#  random_file:: Source of random data used for various cryptographic functions.
+		#
+#		random_file          = /dev/urandom
+
+		#
+		#  check_cert:: Server certificate verification requirements.
+		#
+		#  May be one of:
+		#
+		#  [options="header,autowidth"]
+		#  |===
+		#  | Option | Description
+		#  | `no`   | Server certificate can be signed by any CA or be self-signed.
+		#  | `yes`  | Server certificate must be issued by one of the trusted CAs.
+		#  |===
+		#
+		#  Default is `yes`
+		#
+#		check_cert = no
+
+		#
+		# curl_ssl_opt:: To what degree curl should use ssl
+		#
+		# May be one of:
+		#
+		#  [options="header,autowidth"]
+                #  |===
+                #  | Option | Description
+                #  | `all`   | Require ssl validation to accept login
+                #  | `none`  | Do not try to establish ssl connection
+                #  | `try`  | Try to establish ssl connection, continue even if it cannot
+                #  |===
+                #
+                #  default is 'try'
+                #
+#		curl_ssl_opt = try
+
+		#
+		#  check_cert_cn:: Server certificate CN verification requirements.
+		#
+		#  May be one of:
+		#
+		#  [options="header,autowidth"]
+		#  |===
+		#  | Option | Description
+		#  | `no`   | Server certificate CN can be any value.
+		#  | `yes`  | Server certificate CN matches the host in the URI.
+		#  |===
+		#
+		#  Default is `yes`
+		#
+#		check_cert_cn = no
+
+		#
+		#  extract_cert_attrs:: Extract OIDs from presented certificates as OIDs.
+		#
+		#  Default is `no`
+		#
+#		extract_cert_attrs = no
+	}
+
+	#
+	#  imap_uri:: It will open a connection to the server specified in and send the tls
+	#  request here
+	#
+	imap_uri = "http://127.0.0.1:9090/"
+	#
+	# timeout:: This is how long freeradius will wait for a response 
```

Comment:
`Number of milliseconds FreeRADIUS will wait for a response` - at least, I presume it's milliseconds.

Reply:
We normalised it all across the modules, so should be a seconds with a fractional component.

Reply:
the actual resolution is milliseconds though